### PR TITLE
Migrate to SymfonyConnect

### PR DESCRIPTION
**Pull request by azjezz**
rebase of #28

### CODE REVIEW

File: Collector/ApiCollector.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace SensioLabs\Bundle\ConnectBundle\Collector;
+namespace SymfonyCorp\Bundle\ConnectBundle\Collector;
 
 use Buzz\Listener\ListenerInterface;
```
Comment (nicolas-grekas) [inline Collector/ApiCollector.php]: how do we decouple from this namespace so that this is compatible with symfonycorp/connect 5.1?

Comment (nicolas-grekas) [reply Collector/ApiCollector.php]: I'm merging to help move forward, but could you please have a look at this @azjezz please?

**Comment by nicolas-grekas** [comment]
Thank you @azjezz.

**Comment by nicolas-grekas** [comment]
Thank you @tucksaun too.

File: Collector/ApiCollector.php (hunk 1 above)
Comment (azjezz) [reply Collector/ApiCollector.php]: sure :+1:

Comment (nicolas-grekas) [reply Collector/ApiCollector.php]: please submit against https://github.com/symfonycorp/connect-bundle

**Pull request merged**